BUILDING_ON_LINUX.md

```diff
@@ -9,10 +9,10 @@ Note on Qt version compatibility: If you are installing Qt from a package manage
 
 ## Arch Linux
 install [chatterino2-git](https://aur.archlinux.org/packages/chatterino2-git/) from the aur or build manually as follows:
-1. `sudo pacman -S qt5-base qt5-multimedia qt5-svg gst-plugins-ugly gst-plugins-good boost rapidjson`
+1. `sudo pacman -S qt5-base qt5-multimedia qt5-svg gst-plugins-ugly gst-plugins-good boost rapidjson pkgconf openssl`
 1. go into project directory
 1. create build folder `mkdir build && cd build`
-1. `qmake .. && make`
+1. `qmake .. && make -j$(nproc)`
```

Comment:
It's very uncommon you add -j flags on a package in my experience.
This will disable you from doing anything else while getting the package.

Reply:
I thought this was for the aur package. mb.